src/core_landice/Registry.xml

```diff
@@ -165,6 +165,42 @@
 	                possible_values=".true. or .false."
 		/>
+		<nml_option name="config_restore_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the restore_damage function."
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_calculate_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the calculate_damage function."
+		            possible_values=".true. or .false."
+                />
+                <nml_option name="config_damage_restore_threshold" type="real" default_value="0.0" units="unitless"
+                        description="It is a value above which the old damage number would be kept from the impact of advection (the damage would not heal). For example, if the damage at some location is 0.83, and we set config_damage_restore_threshold to 0.8, then the damage number there will not fall below 0.83. Otherwise, if the damage is 0.7, then it could be decreased to 0.6 if non-damaged ice is transported from upstream. If we set config_damage_restore_threshold to 0 (default), then it will just not play a role in pratical."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_calving_threshold" type="real" default_value="1.0" units="unitless"
+                        description="It is a value above which the cell would be calved away."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_stiffness_min" type="real" default_value="0.1" units="unitless"
+                        description="When coupling damage to rheology, we do not allow it to fall below this value"
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_rheology_coupling" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we change stiffnessFactor to 1-damage during the run."
+	                possible_values=".true. or .false."
+		/>
+                <nml_option name="config_damage_use_constant_A" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we use constant A (i.e., config_default_flowParamA) in the damage calculation."
+	                possible_values=".true. or .false."
+		/>
+                <nml_option name="config_damage_if_calving_rate" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we use a parameterized calving velocity to connect calving and damage."
+	                possible_values=".true. or .false."
+		/>
+    <nml_option name="config_damagecalvingParameter" type="real" default_value="1.0e-4" units="m/s"
```

Comment:
... and in line with the renaming suggestion from Matt above maybe this would then be more clear if renamed, "config_damage_calving_rate" (since this number only applies if we are specifying a calving rate).

Reply:
> Does this parameter only apply if `config_damage_if_calving_rate` is true? If so, let's add that info to the description.

Yes. Info added.

Reply:
> ... and in line with the renaming suggestion from Matt above maybe this would then be more clear if renamed, "config_damage_calving_rate" (since this number only applies if we are specifying a calving rate).

Now I use a different name, config_damage_calving_method. So we can choose to use "calving_rate" method or "threshold" method.